src/chart/main/chart.cpp

```diff
@@ -22,9 +22,9 @@ Chart::Chart() :
 	nextOptions = std::make_shared<Gen::Options>();
 
 	animator->onDraw.attach(
-	    [&](Gen::PlotPtr actPlot)
+	    [&](const Gen::PlotPtr& actPlot)
```

Comment:
I think passing by value and move was a clang-tidy suggestion to optimize out the copy in case of rvalue passed.

Reply:
Yes I know, but the `attach()` is changed, it cannot accept `&&` as arg (and the binary is smaller this way).

Reply:
No warning, it passed the clang-tidy